mw-webapp/src/component/dropdown/dropdownMenuItem/DropdownMenuItem.module.scss

```diff
@@ -1,14 +1,24 @@
 @import "src/styles/variables";
 
 .dropdownMenuItem {
-  padding: $paddingSmall;
+  padding: $paddingSmall $paddingMedium;
+  font-size: $fontSizeMedium;
+  cursor: pointer;
 
-  &:hover {
-    background-color: var(--secondaryBgColor);
-    cursor: pointer;
+  &:not(:last-child) {
+    border-bottom: $primaryBorder solid var(--secondaryStrokeColor);
```

Comment:
On the maket there are using other token-. PLease check or discuss with designers

Reply:
what other token dowe need? Please, explain

Reply:
fixed